cypress.json

```diff
@@ -1,5 +1,4 @@
 {
   "defaultCommandTimeout": 6000,
-  "video": false,
-  "baseUrl": "http://localhost:3000"
```

Comment:
why do we need them to run on different ports? I don't expect we need to run them concurrently, do we?

Reply:
there was an issue with cypress not shutting down the first client (create-react-app) before it attempting to run the second client (persisted-queries) within `acceptance-tests.yml`